Library/ViewModels/MessagesViewModel.swift

```diff
@@ -191,12 +198,16 @@ public final class MessagesViewModel: MessagesViewModelType, MessagesViewModelIn
       .ignoreValues()
 
     // TODO: Call blocking GraphQL mutation
-    self.userBlocked = self.blockUserProperty.signal.map { true }
+    self.userHasBeenBlocked = self.blockUserProperty.signal.map { true }
 
-    self.userBlocked.observeValues { didBlock in
+    self.userHasBeenBlocked.observeValues { didBlock in
       guard didBlock == true else { return }
       NotificationCenter.default.post(.init(name: .ksr_blockedUser))
     }
+
+    /// TODO(MBL-1025): Get isBlocked status from the backend instead.
+    self.participantIsBlocked = self.viewWillAppearProperty.signal
+      .mapConst(true)
```

Comment:
Two things: 

1. Obviously a ``//TODO`` but I would merge this as `false`, since the feature flag defaults to off.
2. Is this redundant with `userHasBeenBlocked`? Why not just emit an initial value of `userHasBeenBlocked` on `viewDidAppear`?

Reply:
That's fair. 
`participantIsBlocked` tells us whether the user is blocked when opening a thread
`userHasBeenBlocked` tells us whether a new request to block someone was successful or not. 

They have different intentions so I opted to keep them separated.

Reply:
Perhaps a naming change would help then - just to throw out the first idea on the top of my head, `participantWasBlockOnLoad` and `didBlockUser`

Reply:
Actually, one more thought here - what is supposed to happen in this case:

1. View messages, no one is blocked
2. You block a user
3. Should reply be disabled? I assume showing the banner is redundant, but perhaps these signals are still kind of linked.